hypothesis-python/tests/nocover/test_simple_numbers.py

```diff
@@ -80,20 +80,20 @@ def test_minimal_small_number_in_large_range():
 
 def test_minimal_small_sum_float_list():
     xs = minimal(lists(floats(), min_size=10), lambda x: sum(x) >= 1.0)
-    assert sum(xs) <= 2.0
+    assert sum(xs) == 1.0
```

Comment:
```suggestion
    assert xs == [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
```

Though using `min_size=5` might make this easier to read.